src/coreclr/jit/simdashwintrinsic.cpp

```diff
@@ -746,41 +756,12 @@ GenTree* Compiler::impSimdAsHWIntrinsicSpecial(NamedIntrinsic       intrinsic,
                 case NI_Vector3_get_Zero:
```

Comment:
Does it handle Vector2 like previously?

Reply:
Yep, it's just one line outside the diff view.

Vector2/3/4 don't have a get_Count property